.github/workflows/publish.yml

```diff
@@ -20,14 +19,24 @@ jobs:
         node-version: '20.x'
         registry-url: 'https://registry.npmjs.org'
 
+    # Determine release tag based on the tag name
+    - name: Determine release tag
+      id: release_tag
+      run: |
+        if [[ "$" == *"-rc"* ]]; then
+          echo "tag=next" >> "$GITHUB_OUTPUT"
+        else
+          echo "tag=latest" >> "$GITHUB_OUTPUT"
+        fi
+
     # Install dependencies (if any) and build your project (if necessary)
     - name: Install dependencies and build
       run: |
         npm install
         npm run build
 
-    # Publish to NPM
+    # Publish to NPM with the appropriate tag
     - name: Publish
-      run: npm publish
+      run: npm publish --tag $
       env:
-        NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+        NODE_AUTH_TOKEN: $
```

Comment:
What happened here? Don't we need the token?

Reply:
updated

Reply:
Twig issues, updated with fixes